test/src/toolbar/toolbar.spec.ts

```diff
@@ -61,56 +65,76 @@ describe('toolbar/toolbar', () => {
 
     });
 
-    describe('#add()', () => {
+    describe('#names()', () => {
+
+      it('should get an ordered list the toolbar item names', () => {
```

Comment:
They are ordered in that they reflect the order of the widgets in the panel layout.